hudi-spark-datasource/hudi-spark/src/main/java/org/apache/hudi/HoodieDataSourceHelpers.java

```diff
@@ -74,6 +74,15 @@ public static List<String> listCommitsSince(HoodieStorage storage, String basePa
         .map(HoodieInstant::getTimestamp).collect(Collectors.toList());
   }
 
+  public static List<String> listCompletionTimeSince(FileSystem fs, String basePath,
```

Comment:
This method is not used.

Reply:
This will be called in integtest script: `docker/demo/sparksql-incremental.commands`